SageMaker/01_introduction.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>I would start the notebook with a 2-3 sentence intro: In this notebook we will ...</p><p> </p><blockquote>You created this role during the notebook creation process.</blockquote><p>You created this role during the notebook creatiion process described in the README.md in <code>SageMaker/README.md</code>.</p><p> </p><p>In the next cell we will check an Amazon S3 bucket exists and create a new one if not. In addition we'll get the SageMaker role and session.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>